rootfs/scheduler/k8s.py

```diff
@@ -255,7 +291,8 @@ def _get_rc_(self, name, namespace):
     def deploy(self, name, image, command, **kwargs):
         logger.debug('deploy {}, img {}, params {}, cmd "{}"'.format(name, image, kwargs, command))
         app_name = kwargs.get('aname', {})
-        app_type = name.split(".")[1]
```

Comment:
this doesn't look like minio secret mounting fixes to me. can we please keep this PR trimmed down to *just* mounting minio secrets?